text/0000-project-groups.md

```diff
@@ -0,0 +1,322 @@
+# Summary
+
+- Formalize project groups as groups dedicated to completing specific projects
```

Comment:
```suggestion
- Formalize project groups as groups dedicated to specific projects
```

I think this helps clarify @spacekookie's point (now in a resolved thread) about them being ephemeral.